tldw_Server_API/tests/DB_Management/test_db_migration_verification.py

```
@@ -0,0 +1,121 @@
+
+def test_verify_migrations_reports_missing_files_when_applied_migrations_exist_and_no_files(tmp_path, monkeypatch):
+    db_path = tmp_path / "app.db"
+    db_path.touch()
+    migrator = DatabaseMigrator(str(db_path), str(tmp_path / "missing"))
+
+    monkeypatch.setattr(
+        migrator,
+        "get_applied_migrations",
+        lambda: [{"version": 1, "checksum": "a"}],
+    )
+    monkeypatch.setattr(migrator, "load_migrations", lambda: [])
```

Ruff (0.15.10)

[warning] Prefer `list` over useless lambda

Replace with `lambda` with `list`

(PIE807)